js/jquery.pickmeup.js

```diff
@@ -681,12 +699,19 @@
 						options.binded.update_date();
 					}
 					// Move current month to the first place
-					options.current.addMonths(Math.floor(options.calendars / 2) - instance_index);
+					options.current.addMonths(align(options) - instance_index);
 				} else {
 					var val	= parseInt(el.text(), 10);
-					options.current.addMonths(instance_index - Math.floor(options.calendars / 2));
+					options.current.addMonths(instance_index - align(options));
 					if (el.hasClass('pmu-not-in-month')) {
 						options.current.addMonths(val > 15 ? -1 : 1);
+						if (options.alignment === parseInt(options.alignment, 10)) {
+							instance_index = instance_index + (val > 15 ? -1 : 1);
+							options.alignment = instance_index;
+						}
+					}
+					if (options.alignment === parseInt(options.alignment, 10)) {
+						options.alignment = instance_index;
```

Comment:
Line 708 is the same condition as earlier. It checks for the existence of an integer offset. This offset is necessary to identify how far over the "current" month sits. 

This block is used to adjust the offset for the move forward/backwards when selecting a date outside of the month. 

Line 710 and 714 are both there to assign the new `instance_index` to the alignment since that is now the offset for the current calendar. It must be updated to maintain the location of the entire set of calendars.